### PR TITLE
Feature/7: Footer

### PR DESCRIPTION
**Pull request by YumeAyasaki**
In this PR, I have:

- Checkout from the feature/8 branch...
- Make a footer component...
- Add icons to the public folder...
- Include said component to the _app.js file, thus making it a theme component...

Note: About the icon, should we use a library for that, or use the font in Figma or just export and include it inside the public folder...?
And the linking thing is not yet available...

localhost_3000_Test - Cá nhân 1 - Microsoft​ Edge 9_16_2022 11_54_37 AM

### CODE REVIEW

File: components/theme/footer.jsx [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+import Image from 'next/image'
+
+import styles from './footer.module.scss'
+
+export default function Footer() {
+  return (
+    <div className={styles.footer}>
+      {/* Title */}
+      <h5 className={styles.title}> 
+        Vườn hoa sứ
```
Comment (PaulHo0501) [inline components/theme/footer.jsx]: Change this to "Vườn Hoa Sứ"

File: components/theme/footer.jsx [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+      </h5>
+      
+      {/* Icon */}
+      <div className={styles.iconContainer}>
+        <div className={styles.icon}><Image src='/icons/facebook.svg' alt='facebook' layout='fill' objectFit='contain'/></div>
```
Comment (PaulHo0501) [inline components/theme/footer.jsx]: Please use the library "[react-icons](https://react-icons.github.io/react-icons/)" for this

File: components/theme/footer.module.scss [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    text-align: center;
+    justify-content: center;
+    
+    color: white;
+
+    .title {
```
Comment (PaulHo0501) [inline components/theme/footer.module.scss]: Apply global styling to these element after changing the styling into classes or mixins

**Review by PaulHo0501 — CHANGES_REQUESTED** [review]
Please check the comments.

**Comment by YumeAyasaki** [comment]
Update PR:
- Rename to Vuon Hoa Su...
- Apply react-icons...
- Edit minor things to match the new global styling...

Note: As you can see, the old module sass file doesn't match well with the new global styling. Any idea for that...?

File: components/theme/footer.module.scss [hunk 4]
```diff
@@ -31,15 +31,15 @@
     }
```
Comment (PaulHo0501) [inline components/theme/footer.module.scss]: Please capitalize the component's name (eg Footer, not footer)

**Review by PaulHo0501 — CHANGES_REQUESTED** [review]

**Comment by YumeAyasaki** [comment]
Update PR:
- Update some name classes...
- Rename footer to Footer...

Note: Do you have any idea regarding the name change commit of VS Code...?

**Comment by PaulHo0501** [comment]
LGTM

**Pull request merged**